player.rb

```diff
@@ -0,0 +1,30 @@
+# Class for a blackjack player to keep track of cards in hand and total value
+class Player
+  attr_reader :hand, :total
+  attr_accessor :hand, :total
+
+  def initialize
+    @hand = []
+    @total = calculate_total
+  end
+
+  def get_card_val(sum, card)
+    # Face cards count as 10
+    if %w(J K Q).include?(card.val)
```

Comment:
Would this be a good opportunity for a case statement? Here's a nice write up on them http://www.skorks.com/2009/08/how-a-ruby-case-statement-works-and-what-you-can-do-with-it/